opencti-platform/opencti-front/lang/en.json

```
@@ -2506,6 +2505,27 @@
   "The tag has been removed": "The tag has been removed",
   "I have read and comply with the above statement": "I have read and comply with the above statement",
   "You need to activate OpenCTI enterprise edition to use this feature.": "You need to activate OpenCTI enterprise edition to use this feature.",
+  "Max Confidence Level": "Max Confidence Level",
+  "This value is overridden for the following entity types": "This value is overridden for the following entity types",
+  "Enable user max confidence level": "Enable user max confidence level",
+  "The user's max confidence level overrides the max confidence that might be set at groups level.": "The user's max confidence level overrides the max confidence that might be set at groups level.",
+  "Important notice: your action is required!": "Important notice: your action is required!",
+  "this blogpost": "this blogpost",
+  "our Slack channel": "our Slack channel",
+  "The following groups require your attention:": "The following groups require your attention:",
+  "and ... more": "and {count} more.",
+  "Open Settings": "Open Settings",
+  "alert_GROUP_WITH_NULL_EFFECTIVE_LEVEL": "Starting with OpenCTI 6.0, a max confidence level will be added to groups and users. It will provide platform administrators with more control on the impact of imported data and prevent any unwanted update on data. Please read {link_blogpost} for more information.\n \nOn this platform, \"max confidence level\" of some groups are currently undefined. Their members will not be able to create any data in version 6.X of the platform. In order to prevent that, make sure all groups have a max confidence level set. More info in {link_blogpost}. Be particularly mindful of users associated to Connectors, Feeds and Streams; you can set individual confidence level to override group confidence level if need be. Join {link_slack} if you need advices.",
```

```suggestion
  "alert_GROUP_WITH_NULL_EFFECTIVE_LEVEL": "Starting with OpenCTI 6.0, a max confidence level will be added to groups and users. It will provide platform administrators with more control on the impact of imported data and prevent any unwanted update on data. Please read {link_blogpost} for more information.\n \nOn this platform, \"max confidence level\" of some groups are currently undefined. Their members will not be able to create any data in version 6.X of the platform. In order to prevent that, make sure that all groups have a max confidence level set. More info in {link_blogpost}. Be particularly mindful of users associated to Connectors, Feeds and Streams; you can set individual confidence level to override group confidence level if needed. Join {link_slack} if you need advices.",
```
Feel free to refuse, "if need be" sound weird to me, but English is not my native language.